jenkins/create_state_county_country_basemap

```diff
@@ -0,0 +1,70 @@
+pipeline {
+  agent {
+        node {
+            label 'team:makerspace'
+        }
+    }
+  stages {
+    stage('Clean Workspace') {
+	    steps{
+		    cleanWs()
+	    }
+	  }
+    stage('Checkout repo and pull from S3') {
+      steps {
+        sh 'wget -O DOIRootCA2.cer http://sslhelp.doi.net/docs/DOIRootCA2.cer'
+         checkout([$class: 'GitSCM',
+                          branches: [[name: "${params.BRANCH_TAG}"]],
```

Comment:
This needs to be defined by a parameter block I think, like here: https://github.com/usgs-makerspace/wbeep-processing/blob/master/jenkins/tippecanoe_create_tiles_Jenkinsfile#L7

Reply:
I did add this, but since we're only deploying to maptiles-prod-website and only pulling the geojson from the s3 bucket at test, so I'm not sure I really need the choice parameter here, but more in the jenkins job itself for selecting which branch of the repo to use